atomic_reactor/plugins/post_rpmqa.py

```diff
@@ -112,10 +94,9 @@ def gather_output_scratch(self):
             tar_archive.extractall(rpmdb_dir)
 
             rpmdb_path = os.path.join(rpmdb_dir, RPMDB_DIR_NAME)
-            rpmdb_packages = os.path.join(rpmdb_path, RPMDB_PACKAGES_NAME)
 
-            if not os.path.exists(rpmdb_packages):
-                self.log.info('%s does not exist in rpmdb', RPMDB_PACKAGES_NAME)
+            if not os.path.exists(rpmdb_path) or not os.listdir(rpmdb_path):
```

Comment:
why are you changing this? this shouldn't be changed

Reply:
from jira - Fedora: recent fedora have changed names of files, make sure that folder is not empty don't check for exact names.

Reply:
ack